spec/system/feedback_spec.rb

```diff
@@ -26,6 +26,22 @@
       end
     end
 
+    context "when on the results page, with end_of_journey_flag enabled", :end_of_journey_flag do
+      it "I can successfully submit freetext feedback" do
+        start_assessment
+        fill_in_forms_until(:check_answers)
+        click_on "Submit"
+        expect(page).to have_content("Give feedback on this page")
+        click_on "Give feedback on this page"
+        fill_in "freetext-input-field", with: "some feedback!"
+        click_on "Send"
+        expect(page).to have_content("Thank you for your feedback")
+
+        # Could not get this line of the test to work
+        # expect(FreetextFeedback.find_by(text: "some feedback!", page: "results_show", level_of_help: "certificated")).not_to be_nil
```

Comment:
I suspect it's because the `page` is `show_results` not `results_show` - the default format is action name first then controller name after.